lib/material/material_gui.flow

```diff
@@ -1468,4 +1472,200 @@ MOverlayT(margin : Transform<double>, radius : Transform<double>, style : Transf
 		\sz -> MGetFocusGroup(\parent -> TFrameT(margin, radius, fselectLift(style, \s -> mgraphicsStyle2tgraphicsStyle(parent, s)), sz)),
 		true
 	)
+}
+
+MSegmentedButtons(
+	items : [Pair<Pair<Maybe<MText>, Maybe<MIcon>>, Pair<[MButtonState], DynamicBehaviour<bool>>>],
+	multiSelect : bool,
+	iconsOnly : bool,
+	size : WidthHeight,
```

Comment:
Do not require size without strict need. If you want to support an element as a filler, better to inspect available